Assets/Hathora/Core/Scripts/Editor/Server/ConfigStyle/PostAuth/HathoraConfigPostAuthBodyBuildUI.cs

```diff
@@ -219,9 +219,13 @@ private void onOpenDockerfileBtnClick()
         private void generateDockerfileIfNotExists(HathoraServerPaths _paths)
         {
             bool dockerfileExists = HathoraServerBuild.CheckIfDockerfileExists(_paths);
-            
+
             if (!dockerfileExists)
-                HathoraDocker.GenerateDockerFileStr(_paths);
+            {
+                // TODO: Add UI opts within HathoraServerConfig.Build >>
+                HathoraDocker.GeneratePhotonFusionDockerfileStr(_paths);
```

Comment:
I don't like this pattern because this sample game should show users our recommended path to have them use Hathora for their game. This change essentially us recommending that they should make changes to our dockerfile generation code.

Can we just have an update Dockerfile that they can use and then make it clear in the readme guide that they should uncheck "Overwrite dockerfile"